src/components/companies/users/form.vue

```diff
@@ -119,7 +136,7 @@
                     Cancel
                 </button>
                 <button
-                    :disabled="isLoading || !hasChanged"
```

Comment:
Why did you remove the `!hasChanged` variable?

Reply:
we have no validation for the new field `user_active` because is not required it can be `false|true` so hasChanged not change if I just change this user_active.